kie-server-parent/kie-server-remote/kie-server-rest/kie-server-rest-drools/src/main/java/org/kie/server/remote/rest/drools/CommandResource.java

```diff
@@ -15,6 +15,12 @@
 
 package org.kie.server.remote.rest.drools;
 
+import static org.kie.server.remote.rest.common.util.RestUtils.buildConversationIdHeader;
+import static org.kie.server.remote.rest.common.util.RestUtils.createResponse;
+import static org.kie.server.remote.rest.common.util.RestUtils.getClassType;
+import static org.kie.server.remote.rest.common.util.RestUtils.getContentType;
+import static org.kie.server.remote.rest.common.util.RestUtils.getVariant;
```

Comment:
Static imports should be at the bottom.

Reply:
OK, I just realized the rest of the code has static imports first, so let's keep it this way for the sake of consistency.

We should either apply KIE Java Conventions everywhere or accept that different repos/projects follow different conventions, in which case those conventions should be declared, ready to be imported to mainstream IDEs and automatically **checked**.